### ISSUE TITLE
Support for editing requests

### ISSUE THREAD

**Issue by hansmbakker** [SvenEV/WinSwag#38]
It would be cool if WinSwag would have an editor to manually create and edit requests (like [Postman](https://www.getpostman.com/)) instead of having to use a swagger file to import them.

Also it would be cool if WinSwag could help generating Swagger code from requests and resulting responses.

**Comment by SvenEV** [comment]
I agree that creating requests without swagger files would be very useful, but at the moment the focus is on Swagger support. WinSwag aims to be an alternative to Swagger UI, not a full replacement for Postman.
